website/docs/components/table/partials/code/component-api.md

```diff
@@ -100,21 +103,19 @@ If the `Th` component is passed as the first cell of a table body row, `scope="r
   <C.Property @name="width" @type="string" @valueNote="Any valid CSS">
     If set, determines the column’s width.
   </C.Property>
+  <C.Property @name="yield">
+    Elements passed as children of this component are yielded inside the `<th>` element.
+  </C.Property>
   <C.Property @name="...attributes">
     This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
   </C.Property>
 </Doc::ComponentApi>
 
 ### Table::ThSort
 
-Note: This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of the `ThSort` element.
```

Comment:
@MelSumner I have removed this entirely, because it doesn't make sense in the context of the `ThShort` (just below, we tell the user the component supports an `onClick` event handler) plus this component should **not** have interactive elements as child.